.github/workflows/check_bot_author.yml

```
@@ -18,20 +18,54 @@
     steps:
+      - uses: actions/checkout@v4.2.2
+
+      - name: Load bot patterns
+        id: load-patterns
+        uses: ./.github/actions/load-bot-patterns
+
       - name: Check if PR author is a bot
         id: check-bot
         uses: actions/github-script@v7.0.1
+        env:
+          BOTPATTERNS_JSON: ${{ steps.load-patterns.outputs.bot-patterns-json }}
```

actionlint (1.7.7)

property "bot-patterns-json" is not defined in object type {bot-patterns-script: string}

(expression)